### PR TITLE
Budgets with colors

### PR DESCRIPTION
**Pull request by filipwicha**
działa lepiej niz powinno

### CODE REVIEW

File: .idea/render.experimental.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wiktorowski211) [inline .idea/render.experimental.xml:1]: wtf

File: app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt [hunk 2]
```diff
@@ -23,6 +27,9 @@ class CreateBudgetActivity : BaseActivity() {
         fun newIntent(context: Context) = Intent(context, CreateBudgetActivity::class.java)
     }
 
+
```
Comment (wiktorowski211) [inline app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt]: Remove spaces

File: app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt [hunk 3]
```diff
@@ -34,6 +41,37 @@ class CreateBudgetActivity : BaseActivity() {
 
         create_budget_activity_toolbar.inflateMenu(R.menu.create_budget_menu)
 
+
```
Comment (wiktorowski211) [inline app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt]: Remove spaces

File: app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt [hunk 4]
```diff
@@ -34,6 +41,37 @@ class CreateBudgetActivity : BaseActivity() {
...
+                    true
+                } else false
+            } catch (e: Exception) {
+                Log.wtf("zxc", e.message)
+                false
+            }
+        }
+
+
+
```
Comment (wiktorowski211) [inline app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt]: Remove spaces

File: app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt [hunk 5]
```diff
@@ -67,3 +105,4 @@ class CreateBudgetActivity : BaseActivity() {
         abstract fun provideActivity(activity: CreateBudgetActivity): Activity
     }
 }
+
```
Comment (wiktorowski211) [inline app/src/main/java/com/siano/view/createBudget/CreateBudgetActivity.kt:104]: Remove spaces

File: app/src/main/java/com/siano/view/createBudget/CreateBudgetPresenter.kt [hunk 6]
```diff
@@ -16,8 +16,9 @@ class CreateBudgetPresenter @Inject constructor(
     val budgetsDao: BudgetDao,
     @UiScheduler uiScheduler: Scheduler
 ) {
+
     private val budgetNameSubject = BehaviorSubject.createDefault("")
-    private val budgetColorSubject = BehaviorSubject.createDefault("")
+    private val budgetColorSubject = BehaviorSubject.createDefault("#333333")
```
Comment (wiktorowski211) [inline app/src/main/java/com/siano/view/createBudget/CreateBudgetPresenter.kt:21]: Are you sure we want default color?

File: app/src/main/res/layout/activity_create_budget.xml [hunk 7]
```diff
@@ -41,31 +41,52 @@
...
                     android:layout_height="wrap_content"
-                    android:layout_gravity="center_horizontal">
+                    android:layout_marginTop="15dp">
 
                 <EditText
-                        android:id="@+id/create_budget_name"
+                        android:id="@+id/create_budget_color"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:hint="@string/create_budget_name"
```
Comment (wiktorowski211) [inline app/src/main/res/layout/activity_create_budget.xml:50]: Why color is before name?

File: app/src/main/res/layout/activity_create_budget.xml [hunk 8]
```diff
@@ -41,31 +41,52 @@
...
+
+                <TextView
+                        android:id="@+id/under_color_picker_text"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/touch_to_choose"/>
+
+            </LinearLayout>
+
+
```
Comment (wiktorowski211) [inline app/src/main/res/layout/activity_create_budget.xml]: Remove spaces

File: app/src/main/res/layout/fragment_for_what.xml [hunk 9]
```diff
@@ -33,4 +33,6 @@
 
     </com.google.android.material.textfield.TextInputLayout>
 
+
+
```
Comment (wiktorowski211) [inline app/src/main/res/layout/fragment_for_what.xml]: Remove spaces

File: app/src/main/java/com/siano/view/createBudget/CreateBudgetPresenter.kt (hunk 6 above)
Comment (filipwicha) [reply app/src/main/java/com/siano/view/createBudget/CreateBudgetPresenter.kt:21]: yes

**Pull request closed without merging**